src/annotator/util/buckets.ts

```diff
@@ -70,6 +64,11 @@ export function findClosestOffscreenAnchor(
   anchors: Anchor[],
   direction: 'up' | 'down'
 ): Anchor | null {
+  // Thresholds for how far away an anchor has to be from the top/bottom of
+  // the viewport to be considered off-screen.
+  const BUCKET_TOP_THRESHOLD = 137;
+  const BUCKET_BOTTOM_THRESHOLD = 22;
```

Comment:
The thresholds are still used for deciding which anchor to scroll to when you click on the top/bottom bucket in the bucket bar. This function is used to pick which anchor associated with the top/bottom off-screen bucket to scroll to. Given that we know which bucket is being clicked via the `direction` argument, I think we can simplify this function by just sorting all the anchors by bottom/top position and last/first one. This is an exercise for a future PR though.

Reply:
> Maybe for now at least call out that the thresholds are only relevant when the bucket bar is "mounted" to the sidebar? 

These thresholds still get used when the bucket bar is mounted in a custom container, but they are most likely _wrong_ in that case. This may cause clicking the top/bottom buckets not to behave as expected. I will add a comment to this effect and address in a follow-up.